packages/jest/__tests__/matcher.test.ts

```diff
@@ -46,7 +46,7 @@ describe('toBeAccessible jest a11y matcher', () => {
 
     it.each(a11yConfigParams)('should throw error for dom with a11y issues with config: %#', async (config) => {
         document.body.innerHTML = domWithA11yIssues;
-        expect.assertions(a11yIssuesCount);
+        expect.assertions(a11yIssuesCount + 1);
```

Comment:
why the +1 here now?

Reply:
Good catch :) 
The test has 6 assertions, and a11yIssuesCount was changed from 6 to 5 in this PR to account for the removed `bypass` rule. But this test doesn't actually check for num of a11y issues - think it was a coincidence that the num of issues matched the num of assertions. 

Removed the expectation and added a `hasAssertions` check to the utils method that is using the async calls.